src/panels/lovelace/views/hui-view-background.ts

```diff
@@ -80,7 +77,7 @@ export class HUIViewBackground extends LitElement {
       ) {
         alignment = background.alignment.replace("_", " ");
       }
-      return `${alignment} / ${size} no-repeat url('${background.image}')`;
+      return `${alignment} / ${size} ${background.tile ? "repeat" : "no-repeat"} url('${background.image}')`;
```

Comment:
So the question we had now was if this option should be named `repeat` in code instead of `tile`

Reply:
Depends on how you want to move forward. Tiled backgrounds was the use case and the reason the feature request exists, repeat on the other hand is the more technical term that may not be so obvious to end users. There's also more than just repeat and no-repeat if you're looking at the background-repeat css property and I don't think it can be a mission to just mirror everything css has to offer.

Reply:
That would mean the yaml options would be named:

`background-repeat`
`background-position`
`background-size`

Reply:
Done, the values behind the labels were not aligned to the css values, so they are all changed. (Even spaces in a translation key and it works 🙈)